uni/udb/udb.icn

```diff
@@ -31,9 +36,81 @@ procedure main(argv)
    write("\n   Thank you for using UDB.\n   Goodbye !!!")
 end
 
+procedure multiprog(argv)
+   case argv[1] of {
+      "-dapproxy": { Dapproxy(argv) }
+      "-adapterfactory": { Factory(argv) }
```

Comment:
These will never return, right? Can you please add a comment about that for a future reader?